do/cloud.go

```diff
@@ -63,11 +72,14 @@ func newCloud(config io.Reader) (cloudprovider.Interface, error) {
 	}
 
 	oauthClient := oauth2.NewClient(oauth2.NoContext, tokenSource)
-	doClient := godo.NewClient(oauthClient)
+	doClient, err := godo.New(oauthClient, opts...)
+	if err != nil {
+		return nil, errors.New("failed to create godo client")
```

Comment:
`fmt.Errorf()` to include the error in the return value

Reply:
Sure. Would you like me to do the same for the dropletRegion call?

Reply:
If you don't mind :)